sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificatePolicy.java

```diff
@@ -431,11 +431,11 @@ public CertificatePolicy setCertificateType(String certificateType) {
     /**
      * Set the certificateTransparency value.
      *
-     * @param certificateTransparency the certificateTransparency value to set
+     * @param certificateTransparent the certificateTransparency value to set
      * @return the updated CertificatePolicy object itself.
      */
-    public CertificatePolicy setCertificateTransparency(Boolean certificateTransparency) {
-        this.certificateTransparency = certificateTransparency;
+    public CertificatePolicy setCertificateTransparent(Boolean certificateTransparent) {
+        this.certificateTransparent = certificateTransparent;
```

Comment:
since we are taking in a `Boolean`, should we require NonNull?

Reply:
Boolean means it is nullable.